docs/_data/navigation.yml

```diff
@@ -76,6 +78,8 @@ lnav:
         url: /tutorials/flops-profiler/
       - title: "PyTorch Profiler"
         url: /tutorials/pytorch-profiler/
+      - title: "Autotuning"
```

Comment:
We are trying to maintain the list of tutorials in the left menu in roughly alphabetically order. So Can you move this up like below?
![image](https://user-images.githubusercontent.com/4271600/141540014-eb718511-0611-4341-9b86-647a9a716ebd.png)